end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot

```diff
@@ -27,29 +27,29 @@ Test Navigation to Last Builds
 Store Series Headers
   Wait Until Element is Enabled   ${series_list}
   ${series}=  Get WebElements   ${series_list}
-  
+
   FOR    ${serial}    IN    @{series}
-      
+
       ${header_text}=    Get Line    ${serial.text}    0
       Append To List    ${series_names}    ${header_text}
   END
 
 Test Overview of All Stored Series
     Wait Until Element is Enabled   ${series_list}
-    
+
     FOR    ${header}    IN    @{series_names}
-      Click Element     //h3[.="${header}"]/ancestor-or-self::div[2]/div[2]/div[1]
+      Click Element     //h4[.="${header}"]/ancestor-or-self::section[1]/div[1]/div[1]
```

Comment:
We can surely get a better locator for this element. Let's discuss this together.